lib/mustache/version.rb

```diff
@@ -1,3 +1,3 @@
 class Mustache
-  VERSION = '1.0.0'
+  VERSION = '1.0.1'
```

Comment:
Please don't change the version number.
I'll bump it previous to releases, it makes it easier to track.